### DISCUSSION TITLE
Building Block Explorer and Simulation Explorer are not shown in MoBi Portable V11.2

### DISCUSSION THREAD

**Discussion by GhazalMontaseri** [Open-Systems-Pharmacology/Forum#1537] [Questions & Problems] [answered]
Dear OSPSuite community,
when I execute the MoBi exe file (portable V11.2) Building Block Explorer and Simulation Explorer panels are missing. What I see is:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/30312003/04cb7748-365c-441b-9914-d4b418776f1c)

I tried to open the 'Views' tab and press 'Building Blocks' or even pressing Ctrl+Shift+B or S does not show anything. I also tried to create some building blocks but what I see is just 
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/30312003/89b49191-13c7-4f74-b291-cba386ac83d7)

all BBs separately and not together. Would you please help me with this issue?

Thanks in advance.

**Comment by StephanSchaller** [comment, marked as answer]
Dear @GhazalMontaseri , please check if this solves your issue: https://github.com/Open-Systems-Pharmacology/Forum/discussions/1150

**Comment by PavelBal** [comment]
@Yuri05 I think we need a "FAQ" discussion where questions such as this should be answered.

**Reply by Yuri05** [reply]
yes...

**Reply by StephanSchaller** [reply]
Can we not just create a channel and move / link such often-referred issues there?

**Reply by msevestre** [reply]
Isn't it the how to Channel?
Maybe one called tips and tricks?

**Comment by GhazalMontaseri** [comment]
Thanks @StephanSchaller. It was solved.
